src/common/crc32c_aarch64.c

```diff
@@ -10,25 +11,123 @@ asm(".arch_extension crc");
 #define CRC32CH(crc, value) __asm__("crc32ch %w[c], %w[c], %w[v]":[c]"+r"(crc):[v]"r"(value))
 #define CRC32CB(crc, value) __asm__("crc32cb %w[c], %w[c], %w[v]":[c]"+r"(crc):[v]"r"(value))
 
+#else
+
+#include <arm_acle.h>
+#include <arm_neon.h>
+
```

Comment:
After you changed the check to an assembler based check rather than a compiler based check, please add the an asm(".arch_extension crc+crypto") intrinsic here.